### DISCUSSION TITLE
Does my AR business card  need hosting to be accessible publicly

### DISCUSSION THREAD

**Discussion by Apholabby** [community/community#83228] [Programming Help]
### Body

Hi. I am building a AR business card which I want several clients to be able to access.  But I am having problems concerning hosting. Must I host publicly for clients to be able to access the content or is there any other way to go about it

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by codesmith-emmy** [comment]
I’m not sure I clearly grasp your concerns about hosting your content. But I think [ZapWorks](https://zap.works) may have what you’re looking for. Visit their website.
